apps/condo/domains/banking/schema/PredictTransactionClassificationService.js

```diff
@@ -103,7 +96,7 @@ const PredictTransactionClassificationService = new GQLCustomSchema('PredictTran
                 const result = await response.json()
                 const { prediction: [id] } = result
                 if (!id) {
-                    throw new GQLError(errors.COST_ITEM_NOT_IDENTIFY, context)
+                    return
```

Comment:
We've discussed earlier to keep track transactions, where prediction classifier has failed.
Do we need a log record here or we will query database or use admin UI to select transactions without cost items?